src/modules/findProvinceFromCoordinate/index.ts

```diff
@@ -0,0 +1,74 @@
+import { GeoJSONData } from "./IRGeoJSON";
+
+interface Point {
+	longitude: number;
+	latitude: number;
+}
+
+interface GeoJSONFeature {
+	properties: Record<string, string | undefined>;
+	geometry: {
+		type: string;
+		coordinates: object; // This represents MultiPolygon coordinates
+	};
+}
+
+interface Province {
+	properties: {
+		'name:fa': string;
+		'name:en': string;
+	};
+	geometry: object; 
+}
+
+const provinces = GeoJSONData;
+
+function pointInPolygon(polygon: number[][], point: Point): boolean {
+	let odd = false;
```

Comment:
the variable `odd` doesn't make any sense here ( naming ) 
please change it to something more meaningful 
like `intersect` `isIntersecting`